hvcc/generators/ir2c/static/HvSignalPhasor.h

```diff
@@ -100,9 +100,12 @@ static inline void __hv_phasor_f(SignalPhasor *o, hv_bInf_t bIn, hv_bOutf_t bOut
   *bOut = vsubq_f32(vreinterpretq_f32_u32(vorrq_u32(vshrq_n_u32(pp, 9), vdupq_n_u32(0x3F800000))), vdupq_n_f32(1.0f));
   o->phase = vdupq_n_u32(pp[3]);
 #else // HV_SIMD_NONE
-  const float *p = (const float *)(size_t)((o->phase >> 9) | 0x3F800000);
-  *bOut = *p - 1.0f;
-  o->phase += ((int) (bIn * o->step.f2sc));
+  {
```

Comment:
the additional brackets seem unnecessary I think?

consulted with someone on the union cast and got thumbs up for those ;)

Reply:
Strictly speaking, since this is supposed to be plain C, variable declarations can only be made at the beginning of a block.
I am not sure, how relevant this is with today's C standards, but i am indeed THAT old.

Reply:
Considering the rest of the code is disabled when this case runs, it _is_ the beginning ;)

Reply:
i see

Reply:
So if you remove the additional/unnecessary brackets we can merge this.